core/io/image.cpp

```diff
@@ -3232,6 +3232,12 @@ void Image::_bind_methods() {
 	BIND_ENUM_CONSTANT(COMPRESS_SOURCE_GENERIC);
 	BIND_ENUM_CONSTANT(COMPRESS_SOURCE_SRGB);
 	BIND_ENUM_CONSTANT(COMPRESS_SOURCE_NORMAL);
+
+	BIND_ENUM_CONSTANT(BUMP_TO_NORMAL_ALGO_PARTIAL_DERIVATIVE);
+	BIND_ENUM_CONSTANT(BUMP_TO_NORMAL_ALGO_SOBEL);
```

Comment:
I think these should use `ALGORITHM`, not `ALGO`. We avoid shortening words unless they are really common and universally understood (such as `coords` for `coordinates`).

This is also more consistent with the enum names.

Reply:
Good point, `ALGORITHM` does look better. I think I used `ALGO` because I felt bad about the long names :sweat_smile: 

I changed all occurrences and pushed the change!